phpcs.xml.dist

```diff
@@ -1,16 +1,12 @@
 <?xml version="1.0"?>
-<ruleset name="WPP-PerfLab">
-	<description>WordPress Coding Standards for the Performance Lab Plugin</description>
+<ruleset name="WPP-Performance">
```

Comment:
With this being the root ruleset for all WPP rules, maybe we just name it "WPP"?
```suggestion
<ruleset name="WPP">
```